lib/README.md

```diff
@@ -0,0 +1 @@
+TODO: przesunąć do folderu tutorials, gdy pojawi się faktyczny element folderu lib
```

Comment:
wygląda jak efekt nie umiejętnego podebrać kodu od drugiego zespołu. Ten TODOs jest tu nie potrzebny, ale może Pan opisać do czego potrzebny będzie katalog lib

Reply:
Poprawione. Wpisałem do czego jest przeznaczony ten folder. Jeżeli to co napisałem jest błędne, to poproszę o wyjaśnienie celu istnienia folderu lib.

Reply:
Jest błędnie ... ale to tylko readme ...